### DISCUSSION TITLE
Wyze: Free Web viewing for a couple days

### DISCUSSION THREAD

**Discussion by skilau** [gtxaspec/wz_mini_hacks#370] [General]
Looks like as part of the Wyze week, they are giving away their free web viewing for a couple days.
Of course, all of us are probably using this firmware so we DON'T have to pay to use their live view stuff.

But I was curious, so I went and tried it...

I wonder if others see what I am seeing.

The 3 cameras that I have connected over Ethernet, will simply NOT load a live view.
I just get a spinning circle forever, as it tries to load.

But the 1 cam I still have on Wifi (Floodlight, so I can't really do Ethernet on that), works fine.

I wonder if the Ethernet connection is causing them troubles...?
Anyone else see this?

PS: The live view is here, (no CC needed or anything bad like that):
https://view.wyze.com/live

**Comment by gtxaspec** [comment]
good eye.  i remember it was working before, not sure why its not anymore.
